packages/web-ui/src/ds/atoms/FormField/index.tsx

```diff
@@ -99,52 +99,44 @@ function FormField({
   const formItemId = `${id}-form-item`
   const formDescriptionId = `${id}-form-item-description`
   const formMessageId = `${id}-form-item-message`
+  const input = (
+    <div
+      className={cn('space-y-2 w-full', className)}
+      aria-describedby={
+        !error
+          ? `${formDescriptionId}`
+          : `${formDescriptionId} ${formMessageId}`
+      }
+      aria-invalid={!!error}
+    >
+      {label ? (
+        <Label variant={error ? 'destructive' : 'default'} htmlFor={formItemId}>
+          {label}
+        </Label>
+      ) : null}
+      <FormControl
+        error={error}
+        formItemId={formItemId}
+        formDescriptionId={formDescriptionId}
+        formMessageId={formMessageId}
+      >
+        {children}
+      </FormControl>
 
-  return (
-    <Tooltip
```

Comment:
This trap `onClick` on the parent
```javascript
<div onClick={...}>
  <FormField>...</FormField>
</div>
```

Now it will only trap when `errorStyle` is `tooltip` which is not the default for most of our form inputs